lib/tasks/count_subscribers.rake

```diff
@@ -2,15 +2,14 @@ namespace :query do
   desc "Query how many active subscribers there are to the given subscription slug"
   task :count_subscribers, %i[subscription_list_slug] => :environment do |_t, args|
     slug = args[:subscription_list_slug]
-    subscription_lists = SubscriberList.where(slug: slug)
-    raise "There is no SubscriberList called '#{slug}'" unless subscription_lists.count == 1
+    subscription_list = SubscriberList.find_by!(slug: slug)
+    active_subscriptions = subscription_list.subscriptions.active
 
-    active_subscriptions = subscription_lists.first.subscriptions.where(ended_at: nil)
     puts """
       The SubscriberList '#{slug}' has #{active_subscriptions.count} active subscriptions, of which:
-        - #{active_subscriptions.where(frequency: 0).count} are signed up for Immediate updates
-        - #{active_subscriptions.where(frequency: 1).count} are signed up for Daily updates
-        - #{active_subscriptions.where(frequency: 2).count} are signed up for Weekly updates
+        - #{active_subscriptions.where(frequency: Frequency::IMMEDIATELY).count} are signed up for Immediate updates
```

Comment:
I'm not sure this will work, because `Frequency::IMMEDIATELY` will resolve to the string `"immediately"`, whereas [we want the enum value](https://github.com/alphagov/email-alert-api/blob/d57091a5d33e57d6b69a00a9164dd35279936b8a/app/models/subscription.rb#L7). I think you can just use `:immediately`.